lib/gui/app.js

```diff
@@ -198,6 +198,11 @@ app.config(($provide) => {
   });
 });
 
+app.config(($qProvider) => {
+  // Prevent errors generated by $uibModalInstance.dismiss() unhandled rejections
+  $qProvider.errorOnUnhandledRejections(false);
```

Comment:
Regarding the transition issues, this comment (https://github.com/angular-ui/ui-router/issues/3246#issuecomment-270973117) says it might be due to an application error. Maybe worth digging further?